todo_app/data/trello_items.py

```diff
@@ -2,27 +2,23 @@
 import os
 import dotenv
 
-from todo_app.item import Item
+from todo_app.data.item import Item
 
 
 dotenv.load_dotenv()
 
 def get_items():
     key =  os.getenv("TRELLO_API_KEY")
     token =  os.getenv("TRELLO_API_TOKEN")
-    conn ="trello-proxy.azure-api.net"
+    board_id =  os.getenv("TRELLO_BOARD_ID")
 
 
-    headersList = {
-    "Accept": "*/*",
-    "User-Agent": "Thunder Client (https://www.thunderclient.com)" 
-    }
-
-    response =requests.get ("https://trello-proxy.azure-api.net/1/boards/6575b1e0697c0c91113c2035/lists?key={}&token={}&cards=open&card_fields=id,name".format(key,token),headers= headersList)
+    response =requests.get("https://trello-proxy.azure-api.net/1/boards/{}/lists?key={}&token={}&cards=open".format(board_id, key,token))
 
     response_list =response.json()
     items=[]
     for trello_list in response_list:
+        print(trello_list)
```

Comment:
Minor, but once you're happy that this is working then it's usually a good idea to delete these print statements - for example this one could easily hold sensitive data (e.g. if a user had to-dos like `go see the doctor about <condition>`, or  `email <name> about <business deal>`) which would mean we need to control access to our logs very tightly